playgrounds/petclinic-maven-sb-2/src/main/resources/templates/fragments/selectField.html

```
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+
+<html xmlns:th="https://www.thymeleaf.org">
```

HTMLHint (1.9.2)

[warning] An lang attribute must be present on <html> elements.

(html-lang-require)